### ISSUE TITLE
create index syntax change

### ISSUE THREAD

**Issue by swilly22** [RedisGraph/RedisGraph#1646]
`Neo4j 4.2.4`

query: `create index on :L(v)`

```

This feature is deprecated and will be removed in future versions.

The create index syntax `CREATE INDEX ON :Label(property)` is deprecated, please use `CREATE INDEX FOR (n:Label) ON (n.property)` instead
```

**Comment by swilly22** [comment]
The new create index syntax been merged into the `master` branch

**Comment by ivancea** [comment]
Hi,
About the FOR syntax, is it already working? I'm trying it in a redisgraph 2.8.7 (docker), but it throws the following error:
```
127.0.0.1:6379> GRAPH.QUERY graph "CREATE INDEX FOR (n:Test) ON (n.id)"
(error) errMsg: Invalid input 'F': expected '=' or CREATE INDEX ON line: 1, column: 14, offset: 13 errCtx: CREATE INDEX FOR (n:Test) ON (n.id) errCtxOffset: 13
```

While the old syntax works:
```
127.0.0.1:6379> GRAPH.QUERY graph "CREATE INDEX ON :Test(id)"
1) 1) "Indices created: 1"
   2) "Cached execution: 0"
   3) "Query internal execution time: 0.249300 milliseconds"
```

**Comment by jeffreylovitz** [comment]
Hi @ivancea,

Would you share the output of `info modules`?

Locally, v2.8.7 supports both syntaxes for me:
```
127.0.0.1:6379> GRAPH.QUERY graph "CREATE INDEX FOR (n:Test) ON (n.id)"
1) 1) "Indices created: 1"
   2) "Cached execution: 0"
   3) "Query internal execution time: 0.301578 milliseconds"
```

**Comment by ivancea** [comment]
Hi, thanks for answering @jeffreylovitz:

### `info modules`
```
# Modules
module:name=graph,ver=20807,api=1,filters=0,usedby=[],using=[],options=[]
```
### Redis startup log
```
1:C 18 Jan 2022 22:53:52.511 # oO0OoO0OoO0Oo Redis is starting oO0OoO0OoO0Oo
1:C 18 Jan 2022 22:53:52.511 # Redis version=6.2.5, bits=64, commit=00000000, modified=0, pid=1, just started
1:C 18 Jan 2022 22:53:52.511 # Configuration loaded
1:M 18 Jan 2022 22:53:52.511 * monotonic clock: POSIX clock_gettime
                _._
           _.-``__ ''-._
      _.-``    `.  `_.  ''-._           Redis 6.2.5 (00000000/0) 64 bit
  .-`` .-```.  ```\/    _.,_ ''-._
 (    '      ,       .-`  | `,    )     Running in standalone mode
 |`-._`-...-` __...-.``-._|'` _.-'|     Port: 6379
 |    `-._   `._    /     _.-'    |     PID: 1
  `-._    `-._  `-./  _.-'    _.-'
 |`-._`-._    `-.__.-'    _.-'_.-'|
 |    `-._`-._        _.-'_.-'    |           https://redis.io
  `-._    `-._`-.__.-'_.-'    _.-'
 |`-._`-._    `-.__.-'    _.-'_.-'|
 |    `-._`-._        _.-'_.-'    |
  `-._    `-._`-.__.-'_.-'    _.-'
      `-._    `-.__.-'    _.-'
          `-._        _.-'
              `-.__.-'

1:M 18 Jan 2022 22:53:52.512 # Server initialized
1:M 18 Jan 2022 22:53:52.512 # WARNING overcommit_memory is set to 0! Background save may fail under low memory condition. To fix this issue add 'vm.overcommit_memory = 1' to /etc/sysctl.conf and then reboot or run the command 'sysctl vm.overcommit_memory=1' for this to take effect.
1:M 18 Jan 2022 22:53:52.520 * <graph> Starting up RedisGraph version 2.8.7.
1:M 18 Jan 2022 22:53:52.521 * <graph> Thread pool created, using 4 threads.
1:M 18 Jan 2022 22:53:52.521 * <graph> Maximum number of OpenMP threads set to 4
1:M 18 Jan 2022 22:53:52.521 * Module 'graph' loaded from /usr/lib/redis/modules/redisgraph.so
1:M 18 Jan 2022 22:53:52.521 * Ready to accept connections
```

I tried it launching a simple docker with `docker run --rm -it redislabs/redisgraph:2.8.7`, and then executing:
```
> docker exec <container> redis-cli
> GRAPH.QUERY graph "CREATE INDEX FOR (n:Test) ON (n.id)"
```

I'm not sure where the problem may be

**Comment by swilly22** [comment]
Hi @ivancea, I'm sorry for the confusion the new index creation syntax is only available on our `edge` version, it has yet to be included in our 2.8 version. @jeffreylovitz can we please update the documentation
